python/app/model_publish_dependency_down.py

```diff
@@ -34,7 +34,9 @@ def load_data(self, sg_location):
         """
         # for publishes, sort them by id (e.g. creation date) rather than
         # by update date.
-        SgEntityListingModel.load_data(self, sg_location, sort_field="id")
+        SgEntityListingModel.load_data(
+            self, sg_location, sort_field="id", filters=filters
+        )
```

Comment:
Similar comment as above to initialize filters to the empty list. This would be okay if `SgEntityListingModel.load_data` can handle filters as `None`, but after a quick check, it does not.